Formula/buku.rb

```diff
@@ -209,6 +209,14 @@ def install
     ENV["LC_ALL"] = "en_US.UTF-8"
     ENV["XDG_DATA_HOME"] = "#{testpath}/.local/share"
 
+    expect = nil
+    on_macos do
+      expect = "/usr/bin/expect"
+    end
```

Comment:
```suggestion
    expect = "/usr/bin/expect"
```
This seems cleaner to me, but not a really strong opinion.

Reply:
Done.